src/app/filters/filter-field.ts

```diff
@@ -12,6 +12,7 @@ import { FilterQuery } from './filter-query';
 export class FilterField {
   id?: string;
   placeholder?: string;
+  primaryQueries?: FilterQuery[];
```

Comment:
I'm still not clear as to why we need to provide two options lists? Considering the filter component is performing the filtering, why can we not make an internal copy of the queries property and filter that?